frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/use-summarize-query.ts

```diff
@@ -0,0 +1,120 @@
+import { useCallback, useMemo, useState } from "react";
+
+import * as Lib from "metabase-lib";
+
+export const STAGE_INDEX = -1;
```

Comment:
(I know this is out of scope of this PR, but might be good to add a comment later about what STAGE_INDEX is)